packages/shell-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
I don't know where we need to to leave this comment, so I'll leave it here – in `shell-api.ts`, we fetch the current mongosh version via `require('../package.json').version`. We should replace that with a new method on `EvaluationListener` that we then call from there, so that the mongosh CLI can provide the correct version to the shell-api package (and probably also add an e2e test that `mongosh --eval 'version()'` and `mongosh --version` print the same output).